external-crates/move/crates/move-analyzer/src/bin/move-analyzer.rs

```diff
@@ -33,6 +34,7 @@ const LINT_ALL: &str = "all";
 #[clap(author, version, about)]
 struct Options {}
 
+#[allow(deprecated)]
```

Comment:
Explanation for the deprecated feature [below](https://github.com/MystenLabs/sui/pull/17863/files#r1609145207)